lib/ruby_indexer/test/configuration_test.rb

```diff
@@ -147,17 +147,22 @@ def test_indexable_uris_respect_given_workspace_path
         uris = @config.indexable_uris
         assert(uris.none? { |uri| uri.full_path.start_with?(File.join(dir, "ignore")) })
 
+        part_1, part_2 = Pathname.new(RbConfig::CONFIG["rubylibdir"]).split
```

Comment:
This test started failing because I moved `test-unit` out of the development group. It's one of those default gems that are installed in the alternative directory, so I modified the test to account for those.